test/e2e/framework/framework.go

```diff
@@ -839,6 +840,44 @@ func (f *Framework) LogEvents(t *testing.T, log *logrus.Entry, namespace string,
 	}
 }
 
+// LogPodContainers takes a Pod as an argument and attempts to output the current and previous logs from each container
+// in that Pod It uses the framework's KubeClient to retrieve the logs and outputs them using the provided logger.
+func (f *Framework) LogPodContainers(t *testing.T, pod *corev1.Pod) {
+	log := TestLogger(t)
+	log.WithField("pod", pod.Name).WithField("namespace", pod.Namespace).Info("Logs for Pod:")
+
+	// sub-function so defer will fire on each printLogs, rather than at the end.
+	printLogs := func(container corev1.Container, previous bool) {
+		logOptions := &corev1.PodLogOptions{
+			Container: container.Name,
+			Follow:    false,
+			Previous:  true,
```

Comment:
```suggestion
			Previous:  previous,
```
The previous param seems to only be used on the go logs but not on the pod logs options here

Reply:
🤦🏻 nice catch!